src/ExpressionEvaluator/Core/Source/ExpressionCompiler/ExpressionCompiler.csproj

```diff
@@ -87,10 +87,6 @@
     <DefineConstants>$(DefineConstants);EXPRESSIONCOMPILER</DefineConstants>
   </PropertyGroup>
   <ItemGroup>
```

Comment:
@davkean What race are you concerned about?  If it's the debugger DLLs, I believe they're build by the Concord project, on which every EE project should depend.